src/main/scala/ch/openolitor/buchhaltung/BuchhaltungCommandHandler.scala

```diff
@@ -258,13 +258,7 @@ trait BuchhaltungCommandHandler extends CommandHandler with BuchhaltungDBMapping
                 (sort, aboId) +: (Range(sort + 1, Int.MaxValue) zip (zusatzabosByAbo getOrElse (aboId, List())))
             }
             val zusatzaboOrphansByAbo = zusatzabosByAbo.filterNot { case (p, c) => aboIds.contains(p.id) }
-            val latestUntilNow = {
-              if (abosWithSort isEmpty) {
-                100
-              } else {
-                abosWithSort.last._1
-              }
-            }
+            val latestUntilNow = abosWithSort.map(_._1).lastOption.getOrElse(100)
```

Comment:
move the map after the lastOption else the _._1 is done on the howl list. Sorry my Error.
abosWithSort.lastOption.map(_._1).getOrElse(100)